cloud_infra_center/ocp_upi/roles/configure-installer-and-image/tasks/main.yaml

```diff
@@ -0,0 +1,128 @@
+# =================================================================
+# Licensed Materials - Property of IBM
+#
+# (c) Copyright IBM Corp. 2021 All Rights Reserved
+#
+# US Government Users Restricted Rights - Use, duplication or
+# disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+# =================================================================
+
+---
+# tasks file for configure-installer-and-image
+
+- name: Download OpenShift Installer
+  become: yes
+  get_url:
+    url: https://mirror.openshift.com/pub/openshift-v4/s390x/clients/ocp/{{ openshift_version }}.{{ openshift_minor_version }}/openshift-install-linux.tar.gz
```

Comment:
suggest removing hardcoded URLs to a variables file that can be easily changed should the URLs change in the future.